Network/Socket.hsc

```diff
@@ -221,19 +217,8 @@ module Network.Socket
 
     -- * Low level operations
     -- in case you ever want to get at the underlying file descriptor..
-    , fdSocket
-    , mkSocket
     , setNonBlockIfNeeded
 
-    -- * Internal
-
-    -- | The following are exported ONLY for use in the BSD module and
-    -- should not be used anywhere else.
-
-    , packFamily
-    , unpackFamily
-    , packSocketType
```

Comment:
A number of functions are now not being exported anymore. Should we re-export them here for backwards compatibility? Or are we okay with pushing this much breakage?

Reply:
This is used internally. So, it's not deprecated. Can we put the deprecated pragma only when re-exporting?

Reply:
It seems to me that we cannot deprecate only re-exporting ones.